conans/client/output.py

```diff
@@ -58,18 +58,20 @@ def write(self, data, front=None, back=None, newline=False):
                 data = decode_text(data)  # Keep python 2 compatibility
 
         if self._color and (front or back):
-            color = "%s%s" % (front or '', back or '')
-            end = (Style.RESET_ALL + "\n") if newline else Style.RESET_ALL  # @UndefinedVariable
-            data = "%s%s%s" % (color, data, end)
-        else:
-            if newline:
-                data = "%s\n" % data
-
-        try:
-            self._stream.write(data)
-        except UnicodeError:
-            data = data.encode("utf8").decode("ascii", "ignore")
-            self._stream.write(data)
+            data = "%s%s%s%s" % (front or '', back or '', data, Style.RESET_ALL)
+        if newline:
+            data = "%s\n" % data
+
+        for _ in range(3):
```

Comment:
Ok, this might work and fix the issue, but at least we need some kind of disclaimer for this block of code, maybe a link to the issue, an explanation, (I suppose that a test is impossible)... otherwise anyone having a look at this hack may delete it.

Reply:
@jgsogo  may be it's possible to mock `self._stream.write` in test, so it throws an error 3 times?

Reply:
Added comments and a test patching the stream.write.